Wiki/zh-Tw.js

```diff
@@ -0,0 +1,678 @@
+exports.LANGUAGE = {
+  CODE: "zh-Tw",
+  STANDARD_CODE: "zh-Tw",
+  FULL_CODE: "正體中文",
+  WIKI_CODE: "zh-Tw",
+  CONTRIBUTORS: ["Kami-Sqmf", "iLxlo_"]
+};
+
+exports.TIME = {
+  SECOND: "秒",
+  SECONDS: "秒",
+  MINUTE: "分",
+  MINUTES: "分",
+  HOUR: "小時",
+  HOURS: "小時",
+  DAY: "天",
+  DAYS: "天",
+  WEEK: "週",
+  WEEKS: "週",
+  MONTH: "月",
+  MONTHS: "月",
+  YEAR: "年",
+  YEARS: "年"
+};
+
+exports.PAGE_TITLES = {
+  HOME: "首頁",
+  INVITE_FILO: "邀請 Filo",
+  ANTI_BAD_WORDS: "反壞話",
+  ANTI_EVASION: "反規避",
+  ACTION_LIST: "事項清單",
+  ANTI_INVITES: "反邀請",
+  ANTI_MASS_MENTIONS: "防止提及",
+  ANTI_REPEATED_TEXT: "反重複文字",
+  AUTO_MODERATION: "AUTO MODERATION",
+  ANTI_SPAM: "反洗版",
+  COMMANDS: "指令",
+  DISABLE_COMMANDS: "停用指令",
+  FAREWELLS: "離別訊息",
+  FAREWELLS_VARIABLES: "離別訊息變數",
+  GUILD: "伺服器",
+  GUILD_INVITE: "伺服器邀請",
+  MUTED_ROLE: "禁言身分組",
+  LANGUAGE: "語言",
+  SUPPORTED_LANGUAGES: "支援的語言",
+  LOGGING: "日誌",
+  MEMBER_COUNTER: "人數統計",
+  MEMBER_COUNTER_NICKNAME: "人數統計 (暱稱)",
+  PREFIX: "Prefix",
+  REPORTS: "舉報",
+  WELCOMES: "歡迎訊息",
+  WELCOMES_VARIABLES: "歡迎訊息變數",
+  WELCOME_ROLE: "預設身分組",
+  CLOUDFLARE: "Cloudflare",
+  COMMON_ERRORS: "常見錯誤"
+};
+
+exports.GENERAL = {
+  NOT_AVAILABLE: "不可用的",
+  STEP: "步驟 {{number}}",
+  EXAMPLE: "範例",
+  AND: "和",
+  DO_NOT_INCLUDE_THIS: "當您在打指令時，請勿包含 ``<>``",
+  CHECK_DISCORD_SUPPORT_ARTICLE: "點**[這裡]({{link}})**來查看Discord官方的說明",
+  OPTIONAL: "可填",
+  PERMANENT: "永久",
+  TEMPORAL: "暫時",
+  TEMPORARY: "暫時",
+  PERMANENTLY: "永久",
+  CHANNEL: "頻道l",
+  CHANNEL_ID: "頻道 ID",
+  ROLE: "身分組",
+  ROLE_ID: "身分組 ID",
+  THRESHOLD: "進入點",
+  TIME: "時間",
+  IF_APPLICABLE: "如果適用",
+  LEVEL: "等級",
+  NUMBER: "數字",
+  ACTION: "動作",
+  COMMAND: "指令",
+  MESSAGE: "訊息",
+  USER: "使用者",
+  SERVER: "伺服器",
+  NAME: "名稱"
+};
+
+exports.ABOUT_WIKI = {
+  TITLE: "歡迎來到 Filo 指南百科",
+  FILO_DESCRIPTION: "Filo 是個多功能且強大的 Discord 機器人。 高自訂性、多語言、檢舉系統、日誌、歡迎和離別訊息、人數統計、反邀請、AUTO MODERATION、反規避，以及 +190 個指令",
+  WIKI_DESCRIPTION: " Filo 指南百科的內容會不斷改變以提高其品質 . 本指南涵蓋了一些常見錯誤，以及每個 Filo 指令的指令教學 ",
+  ASK_FOR_HELP: "如果本指南沒有解決您想要的問題，歡迎拜訪我們的 **[Discord 伺服器]({{link}})**",
+  WIKI_KEYS: {
+    TITLE: "指南小提示",
+    DESCRIPTION: "在每個網頁會有一個小提示，它們可以讓您考慮一些重要的資訊。 您可以在下面範例找到這裡提到的一些小提示",
+    SUCCESS_KEY: "說明或建議事項將會顯示在與此類小提示類似的框中",
+    INFO_KEY: "信息或註解將會顯示在與此類小提示類似的框中",
+    WARNING_KEY: "警告將會顯示在與此類小提示類似的框中",
+    DANGER_KEY: "重要訊息將會顯示在與此類小提示類似的框中"
+  },
+  REQUEST_TRANSLATION: {
+    TITLE: "協助翻譯或修正內容！",
+    DESCRIPTION: "如果您需要，您可以要求 更新 或 修改 這份指南的任何主題。無論是為了改進還是為了更正過時或已棄用的用詞" + "\n" + "這份指南擁有著許多語言，都是志願者提供的幫助。如果您想要協助我們翻譯，**[請點擊這裡]({{link}})**"
+  }
+};
+
+exports.HELPFUL_LINKS = {
+  TITLE: "實用連結"
+};
+
+exports.NAVBAR = {
+  HOME: "指南首頁",
```

Comment:
I don't know if this is wrongly translated, if it is well translated you can leave it as it was (the translator does not translate it to me as "Guide Home Page").

Must be "Home".